client/src/components/ProtectedRoute.js

```diff
@@ -4,20 +4,27 @@ import { Route, Redirect } from "react-router-dom";
 
 import AuthService from "../services/auth.service";
 
-const ProtectedRoute = ({ component, ...rest }) => {
-  const isAuthenticated = AuthService.getCurrentUser() !== null;
+const ProtectedRoute = ({ component, onlyAdmin, ...rest }) => {
+  const currentUser = AuthService.getCurrentUser();
+  const isAuthenticated = currentUser !== null;
+  const isAdmin = isAuthenticated && currentUser.admin;
```

Comment:
I think we can keep `isAdmin = currentUser.admin` because if we are not logged in, we will be redirected to `/login`

Reply:
Yep good idea !